toolpathing/FillPathScheduler2d.cs

```diff
@@ -112,7 +133,7 @@ public virtual void AppendPolyline2d(FillPolyline2d curve)
 			}
 
 			Vector2d startPt = curve[iNearest];
-			Builder.AppendTravel(startPt, Settings.RapidTravelSpeed);
+            AppendTravel(currentPos2, startPt);
```

Comment:
Previously there was no check here for whether or not do to a travel with retraction; now this uses the logic in the new method. Not sure if a possible "leaky move" is desired here or not.